### DISCUSSION TITLE
Tenant section is not working

### DISCUSSION THREAD

**Discussion by pankajr03** [microrealestate/microrealestate#131] [Q&A] [answered]
I installed it on my window 10. http://localhost:8080/landlord is working and I can make the use the functionality, but tenant is not working.
Could you please help me on it, why it is not working, and how i can make it working.

**Comment by camelaissani** [comment, marked as answer]
Last commits have fixed the issue and now the tenant app runs properly.
Currently it is still work in progress and not  ready to be used in production.

**Reply by pankajr03** [reply]
thanks

**Comment by flowe14** [comment]
I have install this software base GitHub, all application start i am unbale to create an account to login not sure where to go from here.

does anyone have a docker compose file i can use
